capstone-project/src/main/webapp/map/map.component.js

```diff
@@ -71,12 +72,24 @@ angular.module('map').component('mapComponent', {
             return containerDiv;
         }
 
+        // Sends a marker to the backend for saving.
+        function postMarker(marker) {
```

Comment:
There's inconsistency in this file on defining functions, e.g, buildInfoWindowInput is assigned to a variable. 

Let's be consistent if we can, I like it better the way you defined it here.